devtools/maven/src/main/java/io/quarkus/maven/DevMojo.java

```diff
@@ -114,6 +117,33 @@
     @Parameter(defaultValue = "${preventnoverify}")
     private boolean preventnoverify = false;
 
+    /**
+     * The current build session instance.
+     */
+    @Parameter(defaultValue = "${session}", required = true, readonly = true)
+    private MavenSession session;
```

Comment:
I believe we will have a conflict with another PR retrieving the session also (for a different reason).